motoman/91072e8/91072e8.bug

```diff
@@ -42,4 +42,4 @@ fix:
     - ros-industrial/motoman/motoman_sia20d_support
     - ros-industrial/motoman/motoman_sia5d_support
   languages: null
-  time: 2014-09-02 (11:31)
+  time: 2013-08-08T00:05:20-05:00   # fixed before the issue was reported
```

Comment:
I'm almost certain this isn't correct.

The references issue https://github.com/ros-industrial/motoman/issues/12 was actually closed as `wontfix`, as the packages / directories it refers no longer existed any more at some point.

How do we deal with something which wasn't fixed?